test/phpstan-baseline.neon

```diff
@@ -11,17 +11,12 @@ parameters:
 			path: ../src/Component/PharUpdate/Manifest.php
 
 		-
-			message: "#^Method Deployer\\\\Component\\\\PharUpdate\\\\Version\\\\Builder\\:\\:clearPreRelease\\(\\) should return Deployer\\\\Component\\\\PharUpdate\\\\Version\\\\Builder but return statement is missing\\.$#"
+			message: "#^Property Deployer\\\\Component\\\\PharUpdate\\\\Update\\:\\:\\$file \\(string\\) does not accept null\\.$#"
 			count: 1
-			path: ../src/Component/PharUpdate/Version/Builder.php
+			path: ../src/Component/PharUpdate/Update.php
 
 		-
 			message: "#^Default value of the parameter \\#2 \\$code \\(string\\) of method Deployer\\\\Support\\\\Changelog\\\\ParseException\\:\\:__construct\\(\\) is incompatible with type int\\.$#"
```

Comment:
I guess this one is a phpstan bug.

reported upstream: https://github.com/phpstan/phpstan/issues/4011